sbin/pfmon

```diff
@@ -73,6 +73,8 @@ our $IS_CHILD = 0;
 our %CHILDREN;
 our @TASKS_RUN;
 our $ALARM_RECV = 0;
+our $DEFAULT_TASK_COUNT = 1000;
+our $DEFAULT_TASK_JITTER = 100;
```

Comment:
And then this would be 10 if the above became 100

Reply:
We could configure this easily per task.
The question is do we need that type of granularity?

Reply:
IMO, this would be good enough since it fixes the issue and the leak seemed to happen in the workers that were performing frequent tasks, but if you feel like it, I see no issue in having a per task limit

Reply:
I will add it when we need it.